src/Umbraco.Tests.Integration/Umbraco.Tests.Integration.csproj

```diff
@@ -7,6 +7,10 @@
         <LangVersion>8</LangVersion>
     </PropertyGroup>
 
+    <PropertyGroup Condition=" '$(OS)' == 'Windows_NT' ">
```

Comment:
It might be wise to define properties for all OS options in a props file for all test projects to consume once we reorganize the repo structure. `$(OS)` can't identify OSX so you can use the MSBuild equivalent.

https://docs.microsoft.com/en-us/visualstudio/msbuild/property-functions?view=vs-2019#msbuild-property-functions

https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.osplatform?view=net-5.0#properties

```xml
  <!--Define OS Platform Conditions-->
  <PropertyGroup>
    <IsWindows Condition="$([MSBuild]::IsOSPlatform('Windows'))">true</IsWindows>
    <IsOSX Condition="$([MSBuild]::IsOSPlatform('OSX'))">true</IsOSX>
    <IsLinux Condition="$([MSBuild]::IsOSPlatform('Linux'))">true</IsLinux>
  </PropertyGroup>

  <!--Define compiler processor directives based upon those conditions-->
  <PropertyGroup Condition="'$(IsWindows)'=='true'">
    <DefineConstants>$(DefineConstants);Windows</DefineConstants>
  </PropertyGroup>
  <PropertyGroup Condition="'$(IsOSX)'=='true'">
    <DefineConstants>$(DefineConstants);OSX</DefineConstants>
  </PropertyGroup>
  <PropertyGroup Condition="'$(IsLinux)'=='true'">
    <DefineConstants>$(DefineConstants);Linux</DefineConstants>
  </PropertyGroup>
```

Reply:
Seems reasonable, is Mac OS likely to be required for preprocessor directives? I'm hoping this one won't be required for long, just that one line for LuceneRAMDirectoryFactory

Reply:
In my universe yes, due to differences in handling single precision floating point on different chipsets but I don't know about here. Could be useful for additional stuff though as the tests expand.

Reply:
Maybe when Umbraco see sense and finally adopt EF Core

Reply:
I'm missing some context here :D